app/Models/ComponentGroup.php

```diff
@@ -11,13 +11,14 @@
 
 namespace CachetHQ\Cachet\Models;
 
+use Illuminate\Support\Collection;
```

Comment:
Actually, @ConnorVG you reviewed an outdated diff. Please check the last commit.